colcon_cmake/task/cmake/build.py

```diff
@@ -313,27 +311,15 @@ def _get_make_arguments(self):
     async def _install(self, args, env):
         self.progress('install')
 
-        generator = get_generator(args.build_base)
-        if 'Visual Studio' not in generator:
-            if CMAKE_EXECUTABLE is None:
-                raise RuntimeError("Could not find 'cmake' executable")
-            cmd = [
-                CMAKE_EXECUTABLE, '--build', args.build_base,
-                '--target', 'install']
-            job_args = self._get_make_arguments()
-            if job_args:
-                cmd += ['--'] + job_args
```

Comment:
These three lines have been removed which need to be restored.

Reply:
`job_args` restored when not using a multi-configuration generator. Seems the usage of job args is similarly dependent on the multi-configuration generator check in the build step. The installation code as is may not have worked for Xcode.